lib/bademagic_module/bluetooth/bletoast.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:badgemagic/providers/cardsprovider.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+
+class BleStateToast {
```

Comment:
```suggestion
class ToastUtils {
```

You can shift this to `/lib/bademagic_module/utils`